Woof/Woof/View/OwnerFlow/DetailedPetSitterView/DetailedPetSitterView.swift

```diff
@@ -20,15 +20,44 @@ struct DetailPetSitterView: View {
                 Spacer()
                 PriceLabel(price: viewModel.price)
             }
+
             Divider()
             Text(viewModel.bio)
                 .font(.system(size: AppStyle.FontStyle.footnote.size))
             Divider()
-            Spacer()
+                .padding(.bottom)
+
+            HStack {
+                Text(slotHeaderLabel)
+                Spacer()
+                DatePicker("",
+                           selection: $viewModel.selectedDate,
+                           in: Date.now...,
+                           displayedComponents: .date).labelsHidden()
+                    .font(.system(size: AppStyle.FontStyle.heading.size))
+                    .tint(Color.App.purpleDark)
+                Spacer()
+                Button(searchButtonText) {
+                    Task {
+                        await viewModel.fetchWalkings()
+                    }
+                }
+                .buttonStyle(CapsuleWithWhiteText())
+            }.padding()
+            if viewModel.isLoading {
+                ProgressView()
+            } else {
+                WalkingSlotsView(walkings: $viewModel.walkings)
+            }
```

Comment:
nit: This seems to look more readable:
```suggestion
            }
            .padding()

            if viewModel.isLoading {
                ProgressView()
            } else {
                WalkingSlotsView(walkings: $viewModel.walkings)
            }
```

Reply:
Yes, fixed